src/include/rados/librados.hpp

```diff
@@ -1058,6 +1078,8 @@ namespace librados
     // compound object operations
     int operate(const std::string& oid, ObjectWriteOperation *op);
     int operate(const std::string& oid, ObjectReadOperation *op, bufferlist *pbl);
+    int operate_repair_read(const std::string& oid, librados::ObjectReadOperation *o, bufferlist *pbl,
+		       uint32_t osdid, int32_t epoch, int op_flags);
```

Comment:
`op_flags` is different from `flags` in this context, `flags` is the bitset of `librados::OPERATION_*`, and it's a part of our public API. while `op_flags` is the internal representation of it, which is part of the rados protocol, and is understood by OSD. we use `translate_flags()` to translate from the former to the later. so i'd prefer to keep `op_flags` , so it's more explicit, and less misleading. 

what do you think?